comuni_italiani/autocomplete/apps.py

```diff
@@ -6,59 +6,3 @@
 class ComuniItalianiAutocompleteConfig(AppConfig):
     name = 'comuni_italiani.autocomplete'
     verbose_name = "Comuni Italiani Autocomplete"
-
-    def ready(self):
-        from django.conf import settings
-        import autocomplete_light.shortcuts as autocomplete_light
-        from ..models import Comune, Provincia, Regione, CittaMetropolitana
-        autocomplete_light.register(Comune,
-                                    search_fields=['^name'],
-                                    attrs={
-                                        'placeholder': 'Nome della città',
-                                        'data-autocomplete-minimum-characters': 2,
```

Comment:
i caratteri minimi per la ricerca ora vanno negli [attributi del widget](https://django-autocomplete-light.readthedocs.io/en/master/tutorial.html#passing-options-to-select2) quando decidiamo di usarlo in un form.